### PR TITLE
WIP - Add Pruner Metrics Controller

### PR DESCRIPTION
**Pull request by adambkaplan**

### CODE REVIEW

**Comment by dmage** [comment]
@adambkaplan we'll have a terrible conflict when one of our PRs gets merged.

**Comment by adambkaplan** [comment]
@dmage oh yes we will. Mine is still in smaller commits - I can roll back some of my bigger refactors so that it won't conflict as badly.

File: manifests/09-prometheus-rules.yaml [hunk 1]
```diff
@@ -28,3 +28,26 @@ spec:
...
+        message: |
+          Automatic image pruning is not enabled. Regular pruning of images
+          no longer referenced by ImageStreams is strongly recommended to
+          ensure your cluster remains healthy.
+          
+          To remove this warning, install the image pruner by creating an
+          imagepruner.imageregistry.operator.openshift.io resource with the
+          name `cluster`. Ensure that the `suspend` field is set to `false`.
+    - alert: ImagePruningFailed
+      expr: increase(image_registry_operator_image_pruner_completed_jobs_total{result="failed"}[24h]) > 0
```
Comment (adambkaplan) [inline manifests/09-prometheus-rules.yaml]: fyi @openshift/openshift-team-monitoring - this is a new alert driven by a new metric. The metric is a counter with one label which should only have two values (2 new time series).

**Comment by adambkaplan** [comment]
/close

We don't need a separate controller because job metrics and alerts are generated by `kube-state-metrics`.

**Pull request closed without merging**